src/cunumeric/mapper.cc

```diff
@@ -105,7 +105,8 @@ std::vector<StoreMapping> CuNumericMapper::store_mappings(
     }
     case CUNUMERIC_MATMUL:
     case CUNUMERIC_MATVECMUL:
-    case CUNUMERIC_UNIQUE_REDUCE: {
+    case CUNUMERIC_UNIQUE_REDUCE:
+    case CUNUMERIC_UNZIP_INDICES: {
```

Comment:
based on looking at the implementation of `unzip_indices` below, you don't need any special mapping for it and should fall to the default case at the bottom here.